tenants-attributes-checker/src/main/scala/it/pagopa/interop/tenantsattributeschecker/util/jobs.scala

```diff
@@ -0,0 +1,154 @@
+
+    def createRevokeMessage(expiration: String, expired: String): Message =
+      if (verifier.renewal == PersistentVerificationRenewal.REVOKE_ON_EXPIRATION) Message(expiration, expired)
+      else Message("", "")
+
+    def createRenewalMessage(expiration: String, expired: String): Message =
+      if (verifier.renewal == PersistentVerificationRenewal.AUTOMATIC_RENEWAL) Message(expiration, expired)
+      else Message("", "")
+
+    def createBody(
+      template: MailTemplate,
+      revokeMsg: Message,
+      renewalMsg: Message,
+      attributeName: String,
+      providerName: String,
+      consumerName: String
+    ): String =
+      template.body.interpolate(
+        Map(
+          "ifRevokeExpiration"  -> revokeMsg.expiration,
+          "ifRenewalExpiration" -> renewalMsg.expiration,
+          "ifRevokeExpired"     -> revokeMsg.expired,
+          "ifRenewalExpired"    -> renewalMsg.expired,
+          "attributeName"       -> attributeName,
+          "providerName"        -> providerName,
+          "consumerName"        -> consumerName
+        )
+      )
+
+    def createProviderBody(attributeName: String, providerName: String, consumerName: String): String = {
+      val ifRevoke = createRevokeMessage(
+        "L'attributo sarà revocato e questo potrebbe avere impatti sullo stato di alcune sue richieste di fruizione.",
+        "L'attributo è stato revocato al fruitore e questo potrebbe avere impatti sullo stato di alcune sue richieste di fruizione."
+      )
+
+      val ifRenewal = createRenewalMessage(
+        "Per tua scelta, l'attributo verrà rinnovato automaticamente; non ci sono quindi impatti sulle sue richieste di fruizione.",
+        "Per tua scelta, l'attributo è stato rinnovato automaticamente; non ci sono quindi impatti sulle sue richieste di fruizione."
+      )
+
+      createBody(providerMailTemplate, ifRevoke, ifRenewal, attributeName, providerName, consumerName)
+    }
+
+    def createConsumerBody(attributeName: String, providerName: String, consumerName: String): String = {
+      val ifRevoke = createRevokeMessage(
+        "L'attributo ti verrà revocato e questo potrebbe avere impatti sullo stato di alcune tue richieste di fruizione.",
+        "L'attributo ti è stato revocato e questo potrebbe avere impatti sullo stato di alcune tue richieste di fruizione."
+      )
+
+      val ifRenewal = createRenewalMessage(
+        "Per scelta del fruitore, l'attributo ti verrà rinnovato automaticamente; non ci sono quindi impatti sulle tue richieste di fruizione.",
+        "Per scelta del fruitore, l'attributo è stato rinnovato automaticamente; non ci sono quindi impatti sulle tue richieste di fruizione."
+      )
+
+      createBody(consumerMailTemplate, ifRevoke, ifRenewal, attributeName, providerName, consumerName)
+    }
+
+    val envelope: Future[(InteropEnvelope, InteropEnvelope)] = for {
+      provider           <- tenantProcess.getTenant(verifier.id)
```

Comment:
the convention is (also for other occurrences in the code)
```suggestion
      producer           <- tenantProcess.getTenant(verifier.id)
```